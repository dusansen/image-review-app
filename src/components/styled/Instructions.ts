import styled from 'styled-components';

export const Instructions = styled.p`
  font-size: 0.8em;
  color: #a5a5a5;
`;