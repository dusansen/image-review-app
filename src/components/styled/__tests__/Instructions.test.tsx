import renderer from 'react-test-renderer';
import { Instructions } from '../Instructions';

describe('Instructions', () => {
  it('should render Instructions with styles', () => {
    const tree = renderer.create(<Instructions />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
