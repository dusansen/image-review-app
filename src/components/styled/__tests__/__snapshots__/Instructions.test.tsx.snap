// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Instructions should render Instructions with styles 1`] = `
<p
  className="sc-bdvvtL hHjMAU"
/>
`;
